aggregate_test.go

```diff
@@ -724,6 +724,1418 @@ func Test_Aggregate(t *testing.T) {
+						Component{
+							Name:    "calico",
+							Version: "3.0.2"},
+						Component{
+							Name:    "containerlinux",
+							Version: "1576.5.0"},
+						Component{
+							Name:    "docker",
+							Version: "17.09.0"},
+						Component{
+							Name:    "etcd",
+							Version: "3.3.1"},
+						Component{
+							Name:    "coredns",
+							Version: "1.0.6"},
+						Component{
+							Name:    "kubernetes",
+							Version: "1.9.2"},
+						Component{
+							Name:    "nginx-ingress-controller",
+							Version: "0.11.0"},
+					},
+					Dependencies: []Dependency{},
+					Deprecated:   true,
+					Name:         "kvm-operator",
+					Time:         time.Date(2018, time.March, 1, 2, 57, 0, 0, time.UTC),
+					Version:      "2.1.0",
+					WIP:          false,
+				},
+				Bundle{
+					Changelogs: []Changelog{
+						Changelog{
+							Component:   "containerlinux",
+							Description: "Updated to version 1632.3.0.",
+							Kind:        "changed"},
+						Changelog{
+							Component:   "cloudconfig",
+							Description: "Removed set-ownership-etcd-data-dir.service.",
+							Kind:        "changed"},
+					},
+					Components: []Component{
+						Component{
+							Name:    "calico",
+							Version: "3.0.2"},
+						Component{
+							Name:    "containerlinux",
+							Version: "1632.3.0"},
+						Component{
+							Name:    "docker",
+							Version: "17.09.0"},
+						Component{
+							Name:    "etcd",
+							Version: "3.3.1"},
+						Component{
+							Name:    "coredns",
+							Version: "1.0.6"},
+						Component{
+							Name:    "kubernetes",
+							Version: "1.9.2"},
+						Component{
+							Name:    "nginx-ingress-controller",
+							Version: "0.11.0"},
+					},
+					Dependencies: []Dependency{},
+					Deprecated:   false,
+					Name:         "kvm-operator",
+					Time:         time.Date(2018, time.March, 7, 2, 57, 0, 0, time.UTC),
+					Version:      "2.1.1",
+					WIP:          false,
+				},
+				Bundle{
+					Changelogs: []Changelog{
+						Changelog{
+							Component:   "component",
+							Description: "Put your description here",
+							Kind:        "changed"},
+					},
+					Components: []Component{
+						Component{
+							Name:    "calico",
+							Version: "3.0.2"},
+						Component{
+							Name:    "containerlinux",
+							Version: "1632.3.0"},
+						Component{
+							Name:    "docker",
+							Version: "17.09.0"},
+						Component{
+							Name:    "etcd",
+							Version: "3.3.1"},
+						Component{
+							Name:    "coredns",
+							Version: "1.0.6"},
+						Component{
+							Name:    "kubernetes",
+							Version: "1.9.2"},
+						Component{
+							Name:    "nginx-ingress-controller",
+							Version: "0.11.0"},
+					},
+					Dependencies: []Dependency{},
+					Deprecated:   false,
+					Name:         "kvm-operator",
+					Time:         time.Date(2018, time.March, 13, 12, 30, 0, 0, time.UTC),
+					Version:      "2.1.2",
+					WIP:          true},
+			},
+			ExpectedGroupedBundles: [][]Bundle{
```

Comment:
`ExpectedGroupedBundles` is state with old flannel-operator (only v2: 0.1.0). This should be later corrected to be wanted end result.